.circleci/config.yml

```diff
@@ -18,6 +18,17 @@ jobs:
 
     steps:
       - checkout:
+
+      - run:
+          name: Check-skip
+          command: |
+            export git_log=$(git log --max-count=1 --pretty=format:"%B" | tr "\n" " ")
+            echo "Got commit message:"
+            echo "${git_log}"
+            if [[ -v CIRCLE_PULL_REQUEST ]] && ([[ "$git_log" == *"[skip circle]"* ]] || [[ "$git_log" == *"[circle skip]"* ]]); then
```

Comment:
Could you break this long line using line continuation?

Reply:
Done